deploy.rb

```diff
@@ -78,7 +78,7 @@ def repo_infos
     puts "Deploying #{repo_dir}"
     `bundle install`
     # Comment out where we ask what branch to deploy. We always deploy master.
-    `sed -i '' 's/^\\(ask :branch.*\\)/#\\1/g' config/deploy.rb`
+    `sed -i -e 's/^\\(ask :branch.*\\)/#\\1/g' config/deploy.rb`
```

Comment:
This will work for GNU sed, but doesn't work on osX (BSD sed)

Reply:
@jcoyne as I feared. I'll replace this with a Ruby incantation.

Reply:
Change pushed. Have another look, @jcoyne?